Makefile

```diff
@@ -66,19 +79,20 @@ FILES_TO_JSONNETFMT=$(shell find ./operations/jsonnet ./operations/tempo-mixin -
 ##@ Building
 .PHONY: tempo 	
 tempo: ## Build tempo
-	GO111MODULE=on CGO_ENABLED=0 go build $(GO_OPT) -o ./bin/$(GOOS)/tempo-$(GOARCH) $(BUILD_INFO) ./cmd/tempo 
+	echo $(GOARCH)
+	$(GO_ENV) go build $(GO_OPT) -o ./bin/$(GOOS)/tempo-$(GOARCH) $(BUILD_INFO) ./cmd/tempo
 
 .PHONY: tempo-query
 tempo-query: ## Build tempo-query
-	GO111MODULE=on CGO_ENABLED=0 go build $(GO_OPT) -o ./bin/$(GOOS)/tempo-query-$(GOARCH) $(BUILD_INFO) ./cmd/tempo-query
```

Comment:
afaik CGO_ENABLED is enabled by default

Reply:
Yes `CGO_ENABLED=1` is the default, this is the reason why I'm still disabling it (see the line with `GO_ENV=CGO_ENABLED=0` above)

The talk linked in the PR mentions an alternative method to build portable binaries with by using `-tags netgo,osusergo` and keep cgo enabled. But I don't think that this has any advantages for us, therefore keeping cgo disabled seems the best option for now